DOCUMENTATION.md

```diff
@@ -0,0 +1,121 @@
+D-Bus system bus | `--socket=system-dbus` and `--nosocket=system-dbus` | Toggle | Permit (`--socket=session-dbus`) or prohibit (`--nosocket=session-dbus`) access to the application to the entire system bus.
+Secure Shell agent | `--socket=ssh-auth` and `--nosocket=ssh-auth` | Toggle | Permit (`--socket=ssh-auth`) or prohibit (`--nosocket=ssh-auth`) access to the application to SSH authentications.
+Smart cards | `--socket=pcsc` and `--nosocket=pcsc` | Toggle | Permit (`--socket=pcsc`) or prohibit (`--nosocket=pcsc`) access to the application to smart cards.
+Printing system | `--socket=cups` and `--nosocket=cups` | Toggle | Permit (`--socket=cups`) or prohibit (`--nosocket=cups`) access to the application to printing systems.
+
+#### Device
+
+List of devices available in the sandbox.
+
+Name | `flatpak override` equivalent | Type | Description
+--- | --- | --- | ---
+GPU acceleration | `--device=dri` and `--nodevice=dri` | Toggle | Permit (`--device=dri`) or prohibit (`--nodevice=dri`) access to the application to graphics direct rendering located at `/dev/dri`.
+Virtualization | `--device=kvm` and `--nodevice=kvm` | Toggle | Permit (`--device=kvm`) or prohibit (`--nodevice=kvm`) access to the application to virtualization.
+Shared memory | `--device=shm` and `--nodevice=shm` | Toggle | Permit (`--device=shm`) or prohibit (`--nodevice=shm`) access to the application to shared memory.
+All devices | `--device=all` and `--nodevice=all` | Toggle | Permit (`--device=all`) or prohibit (`--nodevice=all`) access to the application to all devices.
+
+#### Allow
+
+List of features available to the application.
+
+Name | `flatpak override` equivalent | Type | Description
+--- | --- | --- | ---
+Development syscalls | `--allow=devel` and `--disallow=devel` | Toggle | [NEEDS DESCRIPTION]
+Programs from other architectures | `--allow=multiarch` and `--disallow=multiarch` | Toggle | [NEEDS DESCRIPTION]
+Bluetooth | `--allow=bluetooth` and `--disallow=bluetooth` | Toggle | Permit (`--allow=bluetooth`) or prohibit (`--disallow=bluetooth`) access to the application to use bluetooth.
+Controller Area Network bus | `--allow=canbus` and `--disallow=canbus` | Toggle | [NEEDS DESCRIPTION]
+
+#### Filesystem
+
+List of filesystem subsets available to the application.
+
+Name | `flatpak override` equivalent | Type | Description
+--- | --- | --- | ---
+All filesystem files | `--filesystem=host` and `--nofilesystem=host` | Toggle | Permit (`--filesystem=host`) or prohibit (`--nofilesystem=host`) read-write access to the application to the whole filesystem. If it is permitted, the application has read-write access to every file and folder owned by you, the user. The rest will be read-only.
+All system libraries, executables and static data | `--filesystem=host-os` and `--nofilesystem=host-os` | Toggle | [NEEDS DESCRIPTION]
+All system configurations | `--filesystem=host-etc` and `--nofilesystem=host-etc` | Toggle | [NEEDS DESCRIPTION]
```

Comment:
All the `[NEEDS DESCRIPTION]` you see between those lines are descriptions that I do not understand at all. Could you help me here @tchx84?

Reply:
You can find description for those:

1. man page https://docs.flatpak.org/en/latest/flatpak-command-reference.html#flatpak-override or,
2. release notes https://github.com/flatpak/flatpak/releases

e.g, from release notes,  _New filesystem permission "host-etc" and "host-os" give access to system /usr and /etc_.

Reply:
https://github.com/tchx84/Flatseal/pull/136/commits/f438fda2aff7bfa0c0c9d92c8a42f050196c269f